src/vma/dev/ring_bond.cpp

```diff
@@ -358,7 +358,12 @@ void ring_bond::inc_ring_stats(ring_user_id_t id)
 
 bool ring_bond::reclaim_recv_buffers(descq_t *rx_reuse)
 {
-	descq_t buffer_per_ring[m_n_num_resources + 1];
+	descq_t *buffer_per_ring;
+
+	buffer_per_ring = new descq_t[m_n_num_resources + 1];
```

Comment:
@igor-ivanov I prefer we find another solution here.
This flow is fast path and we're calling malloc ('new'). that is against the performance considerations we need to follow.